src/kademlia/dht_storage.cpp

```diff
@@ -268,26 +267,12 @@ namespace
 			torrent_entry* v;
 			if (ti == m_map.end())
 			{
-				// we don't have this torrent, add it
-				// do we need to remove another one first?
-				if (!m_map.empty() && int(m_map.size()) >= m_settings.max_torrents)
+				if (int(m_map.size()) >= m_settings.max_torrents)
```

Comment:
does capping by # of torrents even make sense? a torrent could have a single or thousands of peers announcing.